ui/src/js/views/client.html

```diff
@@ -82,299 +92,349 @@
+                                        </oj-menu-button>
+                                    </template>
+
+                                </oj-table>
+                                <oj-paging-control id="clientNeedsPaging" data='[[clientNeedsDataProvider]]'
+                                    page-size='5' slot='bottom'>
+                                </oj-paging-control>
                             </div>
-                        </oj-dialog>
+        </oj-bind-if>
+    </oj-bind-if>
 
-                        <div class="oj-helper-clearfix">
-                            <oj-button id="addNeedButton" class="oj-sm-float-end" chroming='callToAction' disabled='[[addNeedButtonDisabled]]'
-                                       on-oj-action='[[addNeedButton]]'>
-                                <span>Add Request for Help</span>
-                            </oj-button>
-                        </div>
-                    </div>
+    <oj-dialog style="display:none" id="confirmDeleteDialog" class="oj-sm-12 oj-md-4" dialog-title="Confirm Deletion"
+        cancel-behavior="none">
+        <div slot="body">
+            <p>Are you sure you want to delete this client?</p>
+
+        </div>
+        <div slot="footer">
+            <div class="oj-flex-bar sm-1x-row">
+                <div class="oj-sm-padding-1x-horizontal oj-flex">
+                    <oj-button id="confirmDeleteDialogCancelButton" on-oj-action="[[closeConfirmDeleteDialog]]">
+                        <span>No</span>
+                    </oj-button>
                 </div>
-                <hr />
-                <oj-table id="clientNeedsTable" aria-label="table using array"
-				  columns="{{clientNeedsTableColumns}}"
-						  data="[[clientNeedsDataProvider]]"
-						  translations.msg-initializing="">
-						  <template slot="actionTemplate" data-oj-as="cell">
-							<oj-menu-button chroming="borderless" display='icons' on-oj-action="[[actionListener]]">Action
-								<oj-menu  slot="menu" on-oj-action="[[menuListener]]">
-									<oj-option value="edit" disabled="[[cell.row.need_met === 'Y']]">
-										<span class="oj-ux-ico-check" slot="startIcon"></span>Edit
-									</oj-option>
-									<oj-option value="delete" disabled="[[cell.row.need_met === 'Y']]">
-										<span class="oj-ux-ico-delete-circle" slot="startIcon"></span>Delete
-									</oj-option>
-								</oj-menu>
-							</oj-menu-button>
-						</template>
-		  
-                </oj-table>
-                <oj-paging-control id="clientNeedsPaging" data='[[clientNeedsDataProvider]]'
-                                   page-size='5' slot='bottom'>
-                </oj-paging-control>
+                <div class="oj-sm-padding-1x-horizontal oj-flex">
+                    <oj-button id="confirmDeleteDialogDeleteButton" on-oj-action='[[deleteClient]]'>
+                        <span>Yes</span>
+                    </oj-button>
+                </div>v
```

Comment:
You've got a stray letter "v" on line 416

Reply:
I have removed that 'v'